config/default/kustomization-cluster.yaml

```diff
@@ -0,0 +1,133 @@
+# Adds namespace to all resources.
+namespace: splunk-operator 
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: splunk-operator-
+
+# Labels to add to all resources and selectors.
+commonLabels:
+  name: splunk-operator
+
+bases:
+- ../crd
+- ../rbac
+#- ../persistent-volume
+- ../service
+- ../manager
+# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
```

Comment:
Do we need all these commented lines for later?

Reply:
commented lines are for us to use when we start using webhooks, and patches . these are auto generated code, I am not removing them. if someone is using code to build they can use the webhooks or patches if needed. the code helps in customizing the operator.